smartcontractkit/external-adapters-js
# Feature/37416 add request origin label

## Closes #37416

## Description
Added getClientIp to utils. Updated express POST handler to extract client IP using getClientIp. Updated metrics to save client IP as a label.
......

## Changes

- High level
- changes that
- you made



## Steps to Test

1. Run unit tests
2. Test manually: Run an adapter locally with metrics enabled and check that the following metrics contain the request_origin label:
> http_requests_total
> cache_execution_duration_seconds
> cache_data_get_count
> cache_data_get_values
> cache_data_max_age
> cache_data_set_count
> cache_data_staleness_seconds
> rate_limit_credits_spent_total 

## Quality Assurance
